e2e-tests/portal/cypress/e2e/journeys/checker/return-a-deal-to-maker/return-deal-to-maker.spec.js

```diff
@@ -67,12 +55,10 @@ context('A checker selects to return a deal to maker from the view-contract page
       .successMessageListItem()
       .invoke('text')
       .then((text) => {
-        expect(text.trim()).to.match(/Supply Contract returned to maker./);
+        expect(text.replace(/\s+/g, ' ').trim()).to.match(/Supply Contract returned to maker. View Supply Contract/);
```

Comment:
Can we revert this change, looks like possible merge issue?

Reply:
Done